pkg/kubelet/kubelet.go

```diff
@@ -1521,6 +1545,12 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
 		return fmt.Errorf("no node instance returned for %q", kl.hostname)
 	}
 
+	if kl.configureCBR0 {
```

Comment:
What is it going to happen if configureCBR0 is false? In this PR, you removed the old way to configure cbr0 through configure-vm.sh, but only configure it through kubelet if the flag is true. Shouldn't configure-vm.sh need to check the same flag / env variable?